### PR TITLE
Handle user

### LINKED ISSUE #35: Handle User
- User Entity
- Registration Form 
- Login Form
- Authentification
- DataFixtures

### PR DESCRIPTION
**Pull request by DurandSacha**
close #35 

- User Entity
- Registration form & registration
- Login Form & login
- Authentification
- DataFixtures
- Testing registration and login

### CODE REVIEW

File: src/Security/AppLoginAuthenticator.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        return $credentials['password'];
+    }
+
+    public function onAuthenticationSuccess(Request $request, TokenInterface $token, $providerKey)
+    {
+        if ($targetPath = $this->getTargetPath($request->getSession(), $providerKey)) {
+            return new RedirectResponse($targetPath);
+        }
+
+        // For example : return new RedirectResponse($this->urlGenerator->generate('some_route'));
```
Comment (abdounikarim) [inline src/Security/AppLoginAuthenticator.php]: @DurandSacha did you forget something here ? 😉

**Review by abdounikarim — CHANGES_REQUESTED** [review]

**Comment by abdounikarim** [comment]
@DurandSacha i think you miss some tests :
register -> handle the form submission with incorrect data
register -> handle the form submission with valid data
post -> complete the post login methods (where we go when a submit is done ?)

File: src/Security/AppLoginAuthenticator.php (hunk 1 above)
Comment (DurandSacha) [reply src/Security/AppLoginAuthenticator.php]: the problem is solved, the success of authentication redirects to the home route, thanks

**Comment by DurandSacha** [comment]
I added some tests with registration form.
- verification with correct data
- verification with incorrect data
- verification of redirection on home if success

**Comment by Yohann76** [comment]
Close #35

**Pull request merged**